lib/src/disposable/disposable.dart

```diff
@@ -14,6 +14,7 @@
 
 import 'dart:async';
 import 'dart:collection';
+import 'dart:html';
```

Comment:
Per us looking at this, we've decided that there isn't enough of a need right now to export both `disposable_browser` and `disposable_vm` to allow consumers to use this class without methods that require the html library. Down the line we could make this change if necessary.